src/DocTests.jl

```diff
@@ -276,7 +276,9 @@ end
 function filter_doctests(strings::NTuple{2, AbstractString},
                          doc::Documents.Document, meta::Dict)
     meta_block_filters = get(meta, :DocTestFilters, [])
```

Comment:
Completely unrelated to this PR but just saw this:

```suggestion
    meta_block_filters = get(Vector{Any}, meta, :DocTestFilters)
```

Reply:
If you say so